shell/platform/darwin/macos/framework/Source/FlutterKeyboardManager.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterKeyHandler.h"
+
+#import <Cocoa/Cocoa.h>
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterKeyFinalResponder.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterKeyHandler.h"
+
+/**
+ * A hub that manages how key events are dispatched to various handlers, and
+ * whether the event is propagated to the next responder.
+ *
+ * This class can be added with one or more handlers, as well as zero or more
+ * additional handlers.
+ *
+ * An event that is received by |handleEvent| is first dispatched to *all*
+ * handlers. Each handler responds ascynchronously with a boolean, indicating
+ * whether it handles the event.
+ *
+ * An event that is not handled by any handlers, is then passed to to the first
+ * added additional handlers, which responds synchronously with a boolean,
+ * indicating whether it handles the event. If not, the event is passed to the
+ * next additional responder, and so on.
```

Comment:
additional responder, or handler? It would be good to be consistent when talking about "handlers" or "responders".